modules/xfem/doc/content/modules/xfem/index.md

```diff
@@ -48,7 +48,7 @@ Examples of the types of problems XFEM can solve follow:
   - Phase transition (Stefan) problems
 
 To support interfaces that move based upon calculated growth rates rather than prescribed functions,
-coupling with the [Level Set Module](level_set/index.md) is recommended.
+coupling with the [level set module](level_set/index.md optional=True) is recommended.
```

Comment:
The modules cannot link to each other, because any application can have any combination of modules enabled. So, if an application is trying to create pages and include docs from MOOSE documentation for each module must work by itself, so they don't get errors if something is not enabled.